packages/nestjs-test-utilities/src/lib/builders/module/module.builder.ts

```diff
@@ -57,7 +57,17 @@ export class TestModuleBuilder implements ITestModuleBuilder {
       const module = this.imports[index] as NestJSModule;
       const imports = Reflect.getMetadata("imports", module);
 
-      const targetIndex = imports.findIndex((m: any) => m["name"] === target.name);
+      const targetIndex = imports.findIndex((m) => {
+        if (typeof m === "function") {
+          return m["name"] === target.name
+        } else if (typeof m === "object") {
+          const innerModule = m.module
+          return innerModule ? innerModule["name"] === target.name : false
+        } else {
+          // NOT SUPPORTED
```

Comment:
Can you throw an error here, as that would eliminate false positives if one is not looking :) Something like `${typeof m} is not supported by overrideModule`